sdks/python/build.gradle

```diff
@@ -379,3 +405,21 @@ task buildSnapshot() {
   dependsOn 'sdist'
   dependsOn 'depSnapshot'
 }
+
+project.task('createProcessWorker') {
+  dependsOn ':beam-sdks-python-container:build'
+  dependsOn 'setupVirtualenv'
+  def sdkWorkerFile = file("${project.buildDir}/sdk_worker.sh")
+  def workerScript = "${project(":beam-sdks-python-container:").buildDir.absolutePath}/target/launcher/linux_amd64/boot"
+  def sdkWorkerFileCode = "sh -c \". ${envdir}/bin/activate && ${workerScript} \$* \""
```

Comment:
Why do we create this script here?

Reply:
The command can be supplied to `environment_config` directly.

Reply:
I suppose we want to resolve the paths dynamically based on the build directory. For the sake of simplicity, we could also directly write this to environment_config and use relative paths. IMHO ok to leave it like it is.

Reply:
The worker script activates a virtual env and executes boot.go in a new shell where the arguments are passed only to boot.go.
There are 2 challenges to express it as a single command
1. Its really complex to express this multilevel command with right escaping.
2. The command might not be executed correctly as some executor requires the executable to be an executable and not a shell expression.

Please let me know if you have any suggestion to express it in a better way.

Reply:
Since we are basically doing the same thing with the process environment at Lyft, I still don't understand what the issue is. It is an invocation of sh -c '...' and unless I have missed something the shell command string doesn't require any escaping. Having said that, I don't want to block the rest of the PR if it helps to stabilize the CI job. Let's take it as follow-up, I would prefer we can simplify the build script.